components/Table/Table.vue

```diff
@@ -26,13 +37,40 @@ export default {
     items: {
       type: Array,
       required: true
+    },
+    showExpand: Boolean
+  },
+  data() {
+    return {
+      toggle: [],
+      expanded: []
+    };
+  },
+  computed: {
+    columnSpan() {
+      return this.showExpand ? this.headers.length + 1 : this.headers.length;
```

Comment:
this one will always be the same because this function is used in a `v-if="showExpand"`

Reply:
Done.